assets/js/components/ParticipantsList.tsx

```diff
@@ -30,16 +27,20 @@ type ParticipantProps = {
 };
 
 type ParticipantsListProps = {
-  username: string;
+  clientName: string;
   isModerator: boolean;
   eventChannel: any;
 };
 
-const ModeratorMenu = ({ username, name, isPresenter, eventChannel }: ModeratorMenuProps) => {
+const ModeratorMenu = ({ clientName, name, isPresenter, eventChannel }: ModeratorMenuProps) => {
+  const style = getComputedStyle(document.body);
+  const fontColor = style.getPropertyValue("--font-dark-color");
```

Comment:
I think as we are using in a few places we could move this to a function, which would return `fontColor`.

Reply:
Fixed.